packages/playwright-core/src/server/trace/viewer/traceViewer.ts

```diff
@@ -143,12 +143,12 @@ export async function installRootRedirect(server: HttpServer, traceUrls: string[
     response.statusCode = 302;
     response.setHeader('Location', urlPath);
 
-    if (process.env.OPENAI_API_KEY)
-      response.appendHeader('Set-Cookie', `openai_api_key=${process.env.OPENAI_API_KEY}`);
+    if (process.env.EXPERIMENTAL_OPENAI_API_KEY)
```

Comment:
My thoughts were that we would use the "final" envvar name, so I was thinking `PW_OPENAI_API_KEY`, but this is fine too.